kotest-property/src/commonMain/kotlin/io/kotest/property/Gen.kt

```diff
@@ -24,7 +26,7 @@ import io.kotest.property.arbitrary.of
 sealed class Gen<out A> {
 
    fun generate(rs: RandomSource): Sequence<Sample<A>> = when (this) {
-      is Arb -> this.edgecases().asSequence().map { Sample(it) } + this.samples(rs)
+      is Arb -> this.edges().values(rs).asSequence().map { Sample(it) } + this.samples(rs)
```

Comment:
here we call `.edges().values(rs)` providing a random source `rs` to seed those `EdgeCases.Random`